connect/src/spirc.rs

```diff
@@ -471,6 +421,68 @@ impl SpircTask {
         self.session.dealer().close().await;
     }
 
+    async fn handle_resolve_context(&mut self, context_uri: String) {
+        if context_uri.contains("spotify:show:") || context_uri.contains("spotify:episode:") {
+            return; // not supported by apollo stations
```

Comment:
Something to return an Err for?

Reply:
I'm currently working at the autoplay portion and how to handle it. And will probably do exactly that for the method